NEWS.md

```diff
@@ -1,5 +1,9 @@
 # polars (development version)
 
+## What's changed
+
+* Base R functions work on DataFrame and LazyFrame objects via S3 methods: as.data.frame, as.matrix, dim, head, length, max, mean, median, min, na.omit, names, sum, tail, unique, ncol, nrow (#104 @vincentarelbundock).
```

Comment:
Maybe typo? (104 -> 107)

Because #107 is included in 0.5.0, could you move this item to 0.5.0?

Reply:
Is it included? AFAICT from the commit history, the bump occurred before the merge: https://github.com/pola-rs/r-polars/commits/main

Reply:
@sorhawell was tagged after merging #107.
https://github.com/pola-rs/r-polars/commit/e9d98de423f69a0f94c7b46504741cdc587b389c is tagged as 0.5.0

Reply:
Ah yes, I see now. Thanks for catching this and clearing things up!